src/tests/flyer.test.ts

```diff
@@ -202,4 +202,43 @@ describe('getTrending tests', () => {
     const getFlyersResponse = await FlyerRepo.getTrendingFlyers();
     expect(getFlyersResponse).toHaveLength(5);
   });
+
+  describe('getFlyersByCategorySlug tests', () => {
+    test('getFlyersByCategorySlug - no flyers', async () => {
+      const flyers = await FlyerFactory.create(2);
+      await FlyerModel.insertMany(flyers);
+
+      const getFlyersResponse = await FlyerRepo.getFlyersByCategorySlug(Math.random().toString());
+      expect(getFlyersResponse).toHaveLength(0);
+    });
+
+    test('getFlyersByCategorySlug - 1 flyer returned, multiple flyers stored', async () => {
```

Comment:
Once again, let's make this more descriptive -- here we're trying to find a flyer by slug after inserting a bunch of flyers with only one of them having the expected slug, so maybe something like "getFlyersByCategorySlug - query flyer with existing slug"